sentry_sdk/integrations/starlite.py

```diff
@@ -90,43 +94,43 @@ def injection_wrapper(self: "Starlite", *args: "Any", **kwargs: "Any") -> None:
         )
 
         SentryStarliteASGIMiddleware.__call__ = SentryStarliteASGIMiddleware._run_asgi3  # type: ignore
-        middleware = kwargs.pop("middleware", None) or []
+        middleware = kwargs.get("middleware") or []
```

Comment:
This changes behavior as with `pop` the middleware is removed from the kwargs, but with `get` it will still be in the kwargs. 

Is this intentional?

Reply:
I think it is OK because the next line sets kwargs["middleware"] unconditionally.

Reply:
oh, true! :-) thanks!